docs/lib/DEBUGGING.md

```diff
@@ -40,13 +40,21 @@ $ mbed compile -m ${your_target} -t GCC_ARM -c --profile mbed-os/tools/profiles/
 
 The `--profile mbed-os/tools/profiles/debug.json` option ensures that the build system enables debug symbols and disables optimizations. In addition, it ensures the selection of the debug build of uVisor, which enables the uVisor runtime messages.
 
-Now start the GDB server. This step changes depending on which debugger you are using. If you are using a J-Link debugger, run:
+Now start the GDB server. This step changes depending on which debugger you are using.
 
-```bash
-$ JLinkGDBServer -device ${device_name} -if ${interface}
-```
+* If you are using a J-Link debugger, run:
```

Comment:
i'm not the grammar guru, but i think  `in case` instead of `if` better suites here.

Reply:
I disagree. I think "if" is the better option here.

Reply:
I also prefer "if"